lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1309,6 +1309,8 @@ class TestVideoDescriptorStudentViewJson(TestCase):
     TEST_YOUTUBE_ID = 'test_youtube_id'
     TEST_YOUTUBE_EXPECTED_URL = 'https://www.youtube.com/watch?v=test_youtube_id'
 
+    maxDiff = None
```

Comment:
Don't forget to remove this from the final ?

Reply:
Thanks for catching that -- removed with https://github.com/edx/edx-platform/pull/16590/commits/f1133d4c3c617db76fcbb13b9bf611e4f1fb1715#diff-f92d86ab3c699e6b57869a08b0b352bbL1312